internal/repositories/app.go

```diff
@@ -39,7 +44,7 @@ func (ar AppRepository) IsExistDeploymentNameInDB(deploymentName string) (bool,
 func (ar AppRepository) SaveHashData(allHashData []*api.HashData, deploymentData *models.DeploymentData) error {
 	tx, err := ar.db.Begin()
 	if err != nil {
-		ar.logger.Error("err while saving data in database ", err)
+		ar.logger.Error("err while initiating a transaction to save data in database ", err)
```

Comment:
Maybe "open transaction error"? And logger.WithError(...)
Anyway, it seems that suggested correction is not correct. :)

Reply:
> Maybe "open transaction error"? And logger.WithError(...) Anyway, it seems that suggested correction is not correct. :)

Ok, that seems better.